app/src/main/java/fr/free/nrw/commons/utils/ViewUtil.java

```diff
@@ -28,6 +28,14 @@ public static void showShortSnackbar(View view, int messageResourceId) {
         ExecutorUtils.uiExecutor().execute(() -> Snackbar.make(view, messageResourceId, Snackbar.LENGTH_SHORT).show());
     }
 
+    public static void showLongSnackbar(View view, String message) {
+        if (view.getContext() == null) {
+            return;
+        }
+
+        ExecutorUtils.uiExecutor().execute(() -> Snackbar.make(view, message, Snackbar.LENGTH_LONG).show());
+    }
+
     public static void showLongToast(Context context, String text) {
         if (context == null) {
             return;
```

Comment:
@ShridharGoel You can remove showLongToast method if it isn't used in any other place.

Reply:
@ShridharGoel Can you update this PR accordingly so that we can merge this.

Reply:
@ujjwalagrawal17 Yes, sure.

Reply:
showLongToast is now used in other places so should not be removed in this PR